corehq/pillows/case.py

```diff
@@ -98,15 +98,8 @@ def get_case_pillow(
       - :py:class:`corehq.messaging.pillow.CaseMessagingSyncProcessor`
     """
     if topics:
-        expected_topics = set(CASE_TOPICS + (DEMO_CASE_SQL,))  # the demo case sql topic is only used here
-        assert set(topics).issubset(expected_topics), "This is a pillow to process cases only"
-    else:
-        # if running a demo case pillow, no need for the case pillow process to consume changes from the demo topic
-        if settings.RUN_DEMO_CASE_PILLOW:
-            topics = CASE_TOPICS
-        else:
-            # drain changes from the demo case sql topic
-            topics = CASE_TOPICS + (DEMO_CASE_SQL,)
+        assert set(topics).issubset(CASE_TOPICS), "This is a pillow to process cases only"
```

Comment:
Will this fail if `topics == [DEMO_CASE_SQL]`?

Reply:
Yeah it will, sorry rushed that commit.

Reply:
933374001303dc4b7c0ade7d7ae5f0740596f4b3